### PR TITLE
js for trailer

### PR DESCRIPTION
**Pull request by LibbyKnight**

### CODE REVIEW

**Comment by dzimmerman902** [comment]
Looks good to me

**Comment by Nickpopadopolous** [comment]
Looks good.

**Comment by jeannieaux** [comment]
Beautiful javascript queen. <smile>Looks good.

**Comment by AdamJahur** [comment]
fixed the merge conflict

File: assets/javascript/java2.js [hunk 1]
```diff
@@ -89,7 +89,60 @@ $(document).ready(function () {
...
+
+			// }
+
+
+		})
+	
+}
+	trailer();
+
+//  pop-up calendar function
```
Comment (dzimmerman902) [inline assets/javascript/java2.js]: Delete the pop-up calendar function, isn't needed. I found a new pop up calendar.

**Comment by AdamJahur** [comment]
ok i took out the pop up calendar feature

**Pull request merged**